canonicalwebteam/blog/blog_api.py

```diff
@@ -252,9 +252,6 @@ def _apply_image_template(
                     alt="",
                     width=img_width or width,
                     height=img_height or height,
-                    hi_def=True,
```

Comment:
`hi_def` is mandatory, so it needs to be set to False if we want to try opposite value.

`e_sharpen` is not documented at all, so I don't know what it's doing or what value it has, but likely it's not mandatory (as we don't use it in other places it seems)

https://github.com/canonical/canonicalwebteam.image-template?tab=readme-ov-file#parameters